Dockerfile.dapper

```diff
@@ -6,6 +6,6 @@ ENV DAPPER_OUTPUT=${DAPPER_SOURCE}/output
 
 WORKDIR ${DAPPER_SOURCE}
 
-RUN git config --global --add safe.directory ${DAPPER_SOURCE}
+RUN git config --global --add safe.directory *
```

Comment:
Could you explain this change in the commit message?

Reply:
I added a comment above this line so it'll be clear when looking at the file